pages/linux/numactl.md

```diff
@@ -0,0 +1,16 @@
+# numactl
+
+> Control NUMA policy for processes or shared memory.
+> More information: <https://man7.org/linux/man-pages/man8/numactl.8.html>.
+
+- Run command on node 0 with memory allocated on node 0 and 1:
+
+`numactl --cpunodebind={{0}} --membind={{0,1}} -- {{command}} {{command_arguments}}`
+
+- Run command on CPUs (cores) 0-4 and 8-12 of the current cpu-set:
+
+`numactl --physcpubind={{+0-4,8-12}} -- {{command}} {{command_arguments}}`
+
+- Run command with its memory interleaved on all CPUs:
```

Comment:
```suggestion
- Run a command with its memory interleaved on all CPUs:
```

Reply:
I understand, I was referencing the {{command}} with that sentence. So like saying: run mkdir with ...
I feel I still need to get a hold of the correct wording for tldr :D